packages/concerto-core/test/data/aliasing/parent.json

```diff
@@ -0,0 +1,498 @@
+          },
+          "isArray": false,
+          "isOptional": false,
+          "location": {
+            "$class": "concerto.metamodel@1.0.0.Range",
+            "start": {
+              "offset": 127,
+              "line": 8,
+              "column": 3,
+              "$class": "concerto.metamodel@1.0.0.Position"
+            },
+            "end": {
+              "offset": 152,
+              "line": 10,
+              "column": 3,
+              "$class": "concerto.metamodel@1.0.0.Position"
+            }
+          },
+          "decorators": [
+            {
+              "$class": "concerto.metamodel@1.0.0.Decorator",
+              "name": "test",
+              "location": {
+                "$class": "concerto.metamodel@1.0.0.Range",
+                "start": {
+                  "offset": 127,
+                  "line": 8,
+                  "column": 3,
+                  "$class": "concerto.metamodel@1.0.0.Position"
+                },
+                "end": {
+                  "offset": 137,
+                  "line": 8,
+                  "column": 13,
+                  "$class": "concerto.metamodel@1.0.0.Position"
+                }
+              },
+              "arguments": [
+                {
+                  "$class": "concerto.metamodel@1.0.0.DecoratorTypeReference",
+                  "type": {
+                    "$class": "concerto.metamodel@1.0.0.TypeIdentifier",
+                    "name": "Kid"
+                  },
+                  "isArray": false,
+                  "location": {
+                    "$class": "concerto.metamodel@1.0.0.Range",
+                    "start": {
+                      "offset": 133,
+                      "line": 8,
+                      "column": 9,
+                      "$class": "concerto.metamodel@1.0.0.Position"
+                    },
+                    "end": {
+                      "offset": 136,
+                      "line": 8,
+                      "column": 12,
+                      "$class": "concerto.metamodel@1.0.0.Position"
+                    }
+                  }
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "$class": "concerto.metamodel@1.0.0.ObjectProperty",
+          "name": "str",
+          "type": {
+            "$class": "concerto.metamodel@1.0.0.TypeIdentifier",
+            "name": "Str"
+          },
+          "isArray": false,
+          "isOptional": false,
+          "location": {
+            "$class": "concerto.metamodel@1.0.0.Range",
+            "start": {
+              "offset": 152,
+              "line": 10,
+              "column": 3,
+              "$class": "concerto.metamodel@1.0.0.Position"
+            },
+            "end": {
+              "offset": 162,
+              "line": 11,
+              "column": 1,
+              "$class": "concerto.metamodel@1.0.0.Position"
+            }
+          }
+        }
+      ],
+      "location": {
```

Comment:
Could you remove the locations from the ast please?

Reply:
[This](https://github.com/accordproject/concerto/blob/cc54de136c4f01d918182c3b09f9858720c71ece/packages/concerto-core/test/introspect/metamodel.js#L215) test is doing a doing a check on the parsed cto model with the ast.